### PR TITLE
docs: accessing custom props for `NuxtLayout`

### PR DESCRIPTION
**Pull request by justinkekeocha**
I documented how one can access props passed to a NuxtLayout, in the layout

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/3.api/2.components/3.nuxt-layout.md [hunk 1]
```diff
@@ -46,6 +46,25 @@ Please note the layout name is normalized to kebab-case, so if your layout file
   </NuxtLayout>
 </template>
 ```
+## Custom Props/Layout attrs
+
+`NuxtLayout` also accepts custom props that you may need to pass to the layout. These custom props are accessible via `attrs` in the Nuxt app.
```
Comment (danielroe) [inline docs/3.api/2.components/3.nuxt-layout.md:51]:
```suggestion
`NuxtLayout` also accepts any additional props that you may need to pass to the layout. These custom props are then made accessible as attributes.
```

File: docs/3.api/2.components/3.nuxt-layout.md [hunk 2]
```diff
@@ -46,6 +46,25 @@ Please note the layout name is normalized to kebab-case, so if your layout file
...
+  </NuxtLayout>
+</div>
+```
+
+In the above example, the value of `title` will be available using `$attrs.title` in the template or `useAttrs().title` in `<script setup>` at custom.vue.
+
+```vue[layouts/custom.vue]
+<script setup lang="ts">
+  const layoutCustomProps = useAttrs()
+  console.log(layoutCustomProps.title)//I am a custom layout
```
Comment (danielroe) [inline docs/3.api/2.components/3.nuxt-layout.md:65]:
```suggestion
  console.log(layoutCustomProps.title) // I am a custom layout
```

File: docs/3.api/2.components/3.nuxt-layout.md [hunk 3]
```diff
@@ -46,6 +46,25 @@ Please note the layout name is normalized to kebab-case, so if your layout file
   </NuxtLayout>
 </template>
 ```
+## Passing additional props
```
Comment (DamianGlowala) [inline docs/3.api/2.components/3.nuxt-layout.md:49]:
```suggestion
## Passing Additional Props
```

**Review by DamianGlowala — CHANGES_REQUESTED** [review]

**Review by danielroe — APPROVED** [review]

**Comment by danielroe** [comment]
Nice work, thank you ❤️

**Pull request merged**